controllers/dbaas.redhat.com/crunchybridgeinstance_controller.go

```diff
@@ -0,0 +1,311 @@
+/*
+Copyright 2021 Crunchy Data Solutions, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dbaasredhatcom
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	crunchybridgev1alpha1 "github.com/CrunchyData/crunchy-bridge-operator/apis/crunchybridge/v1alpha1"
```

Comment:
Please make/keep includes consistent with (core <line break> 3rd party <line break> org/local/project ) grouping used in the other go files. (one step in the right direction would use `goimports` on save over `gofmt`, as the former is a superset)

Reply:
used goimprot to group the packages.